ext/nnstreamer/tensor_filter/tensor_filter_pytorch_core.cc

```diff
@@ -44,6 +44,7 @@ TorchCore::TorchCore (const char *_model_path)
 {
   model_path = _model_path;
   configured = false;
+  first_run = false;
```

Comment:
Why is it ```false``` at constructor?

If you want to check if init() is called before invoke, make it "tri-state" (uninit-ed / first / second).

Reply:
Changed to `true` at constructor. Actually, this value does not matter, as init() will be called before invoke(), which will set the value.

> If you want to check if init() is called before invoke, make it "tri-state" (uninit-ed / first / second).

As this ext is called by tensor_filter, I think there is no need to worry about invoke() before init().